backend/tests/mockSQL.js

```diff
@@ -0,0 +1,34 @@
+class MockSQL {
```

Comment:
What do you think about setting up another separate Cloud SQL instance for this?

Reply:
As all of the tables will need to be flushed between tests, having a shared database may cause issues if two people want to run tests at once.

Reply:
Good job on the mock @jacoballen4534. Although it might be more helpful to test directly on the sql servers, it would defeat the purpose of unit tests. I think mock tests are the way to go for now.